newrelic_rpm.gemspec

```diff
@@ -56,6 +56,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'minitest-stub-const', '0.6'
   s.add_development_dependency 'mocha', '~> 1.16'
   s.add_development_dependency 'pry' unless ENV['CI']
+  s.add_development_dependency 'rack'
```

Comment:
I feel that it's much better to test web requests with a real `Rack::Request` instead of mocks, and given that the existing tests use the standard project gemspec, `rack` needed to be added as a dev dependency.

Reply:
Neat idea! I wonder where else this approach might be useful.